index.html

```diff
@@ -1,223 +1,271 @@
             </a>
-            <a href="">
-              <img src="/images/twitt.svg" alt="Логотип Twitter">
-            </a>
-          </li>
-        </ul>
-      </div>
-      <div class="pageleft">
-        <ul class="ul2footer">
-          <li>
-            <h3>We Support</h3>
-          </li>
-          <li>
-            <img src="/images/pay.png" alt="Поддерживаемые способы оплаты: Visa, Mastervard">
-          </li>
-        </ul>
+          </div>
+          <div class="col col-6">
+            <nav class="main-nav">
+              <ul class="main-nav-list">
+                <li class="main-nav-list-item">
+                  <a href="">OVERVIEW</a>
+                </li>
+                <li class="main-nav-list-item">
+                  <a href="">FEATURES</a>
+                </li>
+                <li class="main-nav-list-item">
+                  <a href="">TECHNOLOGY</a>
+                </li>
+                <li class="main-nav-list-item">
+                  <a href="">CONTACT</a>
+                </li>
+                <li class="main-nav-list-item">
+                  <a href="">Sign up</a>
+                </li>
+              </ul>
+            </nav>
+          </div>
+          <div class="col col-2 offset-1">
+            <button class="button-login">login</button>
+          </div>
+        </div>
       </div>
-    </section>
-    <p class="ptextfoot">Copyright 2020 CloudBudget</p> 
-  </footer>
-  <script src='https://febasic.github.io/js/form.js'></script>
-</body>
+    </header>
+    <main>
+      <section class="hero-section ">
+        <div class="container">
+          <div class="row">
+            <div class="col col-8 offset-2 text-center">
+              <h1 class="first-title ">CloudBudget</h1>
+            </div>
+          </div>
+          <div class="row">
+            <div class="col col-6 offset-3 text-center">
+                <p class="desccloud">Cloud budget management for everyone. Only €69.95 a Month After a 7 Day Trial of 
+                  Up to €4.99</p>
+                <button class="button-signup">Sign Up</button>
+              </div>
+            </div>
+          </div>
+        </section>
+        <section class="section-video margin-70">
+          <div class="container">
+            <div class="row">
+              <div class="col col-5">
+                <h2 class="title-medium">Easy to Use Cloud Budget Management Software</h2>
+                <p class="text-video">Our software is made so you can access and manage your budget and expenses online at any time from any device. It provides detailed income and expense reports with graphs so you can easilly see your spending patterns and budget at a glance. Read below to find out more.</p>
+              </div>
+              <div class="col col-6 offset-1">
+                <video class="video" src="https://febasic.github.io/media/video.mp4" poster="/images/Video.jpg"></video>
+                <button class="button-play">

```

Comment:
How does screen reader read the button? How do you think?
Please, add aria-label for button. Example: aria-label: play the video.